### DISCUSSION TITLE
Politische Einflussnahme via Blocklisten? Ctnd.

### DISCUSSION THREAD

**Discussion by Nuube** [RPiList/specials#1622] [General]
Da der andere Thread keine weiteren Kommentare zulässt, dass Problem der politischen Einflussnahme aber weiterhin besteht (und damit auch der Diskussionsbedarf), mache ich hier eine weiteren Thread zum Thema auf.

**Comment by lekl7-de** [comment]
Hallo. 

ich habe noch ein paar offene Fragen die ich im ersten Thread nicht klären konnte.

Zunächst einmal finde ich es gut, dass es die Listen gibt und dass sie auch immer gepflegt werden.  

Meine Frage wäre jetzt die Sperrung von Parteiseiten auf der "child-protection" liste.

Ich verstehe das Prinzip diese Seiten für Kinder zu sperren, aber ich verstehe nicht warum man es dann nur so macht, meiner Meinung nach einseitig, also "afd.de" wird gesperrt aber Seiten wie "cdu.de, spd.de, die-linke.de, dkp.de und gruene.de" bleiben frei?

Ich finde, wenn man Webseiten von Parteien sperrt, dann sollte man alle sperren oder keine.


ps. (ich bin nicht so ein AFD Fanboy aber ich finde man sollte fair bleiben)

Gruß

**Comment by RPiList** [comment]
Hi,

würde ich zustimmen. Soweit ich weiß startet der Bundestag demnächst ein Verfahren gegen die AFD bzgl. der Verfassungswidrigkeit. Aber bis dahin sollte sie als "normale" Partei gelten und nicht geblockt werden.

**Comment by Bob-Nixkoenner** [comment]
Würde ich jetzt nicht zustimmen. 
Aber ggf kann mich ja jemand aufklären, ob es noch andere Parteien / Landesverbände von Parteien gibt, die als gesichert Extrem (Links, Rechts... was auch immer) gelten. Wie viele der genannten Parteien cdu spd, die-linke gruene bsw stehen unter Aufsicht vom Verfassungsschutz? Oder anders gefragt, haben die genannten Parteien gemeinsam so viele beaufsichtigte Mitglieder die AFD zusammen? 
Will sagen, gilt etwas nach demokratischen Werten, oder den Werten der Gesellschaft als Extrem (und dies auch begründet) , gehören Kinder davor geschützt. 

Jeder hat natürlich die Möglichkeit dies anders zu sehen, oder andere Werte als ich zu haben, für die gibt es dann aber auch die Möglichkeit die AFD zu whitelisten, oder die genannten Parteien zu blacklisten. 

Fazit: Eine Childprotect Liste schützt Kinder vor Extremen (nach Werten der Gesellschaft in der wir Leben). 

PS, Wer dennoch den Unterschied zwischen AFD und den anderen genannten Parteien nicht kennt, hat natürlich die Möglichkeit beim Verfassungsschutz mal nachzufragen, ob die einen Unterschied kennen. 
Man kann auch mal prüfen, wie viele Mitglieder der genannten Parteien Straftäter sind, ggf wollt Ihr ja eure Kinder zumindest vor Straftätern schützen.

**Reply by mchawk777** [reply]
Der Punkt, das die Child-Protection-Liste mit linken Gesinnungen freundlicher umgeht als mit konservativen/rechten, war für mich so offensichtlich, dass ich die Nutzung dieser Liste nur mit Vorbehalt empfehlen kann.

_Dauerbeispiel_ ist, dass tichyseinglick geblockt wird - correctiv allerdings nicht - obschon beide schön politisch-populistisch mit deutlicher Schlagseite unterwegs sind.
Entweder blockt man Beides oder gar nichts. Ist mir im Prinzip wuppe - aber es sollte wenn gleich gewichtet geschehen.🤷‍♂️

Aufgrund der Situation, dass aktuell viel zu viele Idealisten alles andere als politisch ausgewogen sind, wirst Du letztendlich _im Detail_ immer selbst Hand anlegen müssen.
RPiList wird bei den automatisierten Prozesses auch nicht alles ausgleichen können, was die Lieferanten der Quelldaten so eingeben.

Schätze auch, dass die Pornblock-Listen für mehr Kinderschutz sorgen als die "Child-Protection". 🤷‍♂️

**Reply by joestr** [reply]
> Der Punkt, das die Child-Protection-Liste mit linken Gesinnungen freundlicher umgeht als mit konservativen/rechten

_Herzlich willkommen bei McPolitics Ihre Bestellung bitte?_

_Sie hätten gerne eine komplett objektive Berichterstattung, die auch Hintergrund-Details mit abdeckt?_

_Tut mir leid, die sind zurzeit ausverkauft._

Es ist nun mal der Umstand, dass die zusammengetragenen Listen irgendeiner politischen Ausrichtung folgen. Das ist auch nichts, wofür man sich schämen muss, sondern einfach nur menschlich.

Wir können auch schlecht eine Strichliste führen à la "Eine Webseite von diesem politischen Spektrum blockiert, im Ausgleich dann auch eine von diesem politischen Spektrum blockiert."

So funktioniert das nicht, das wäre wahnsinnig.

Im Endeffekt wird auch nur das blockiert, wo am meisten Unflat herumkommt. Wenn man sich dann allerdings darin wiederfindet, dass laufend Web-Angebote blockiert werden, welche man konsumiert hat ... tja.

**Reply by RPiList** [reply]
> _Dauerbeispiel_ ist, dass tichyseinglick geblockt wird - correctiv allerdings nicht - obschon beide schön politisch-populistisch mit deutlicher Schlagseite unterwegs sind. 
> 

Der Irrtum besteht darin dass Sie und andere Nutzer glauben, dass die Website gesperrt wird/ist weil sie "rechts" oder "links" ist. So kommen Sie dann auch direkt auf die Spur, dass das Leben unfair sei. Aber wie schon im alten Thread dargelegt geht es i.d.R. nicht um "rechts" oder "links" sondern eher darum ob geschwurbelt wird oder nicht.

Wenn Sie meinen, dass Correctiv für Kinder gesperrt werden sollte, erstellen Sie ein Issue und verweisen auf entsprechend kritische Berichte auf der Seite, die Ihr Issue stützen. Schwurbelt Correctiv wird es dann auch in die Liste aufgenommen. Da mache ich mir gar keine Sorgen.

Natürlich kann es sein, dass die Listen nicht die eigene Ansicht wiederspiegeln. Das ist das schöne am Pihole: Sie können die fehlende Seite selbst hinzufügen oder Listen anderer Maintainer einspielen, die Sie für "gerechter" halten.

**Reply by mchawk777** [reply]
>Der Irrtum besteht darin dass Sie und andere Nutzer glauben, dass die Website gesperrt wird/ist weil sie "rechts" oder "links" ist. >So kommen Sie dann auch direkt auf die Spur, dass das Leben unfair sei. Aber wie schon im alten Thread dargelegt geht es i.d.R. >nicht um "rechts" oder "links" sondern eher darum ob geschwurbelt wird oder nicht.

"Rechts oder links" - "geschwurbel oder nicht" - vollkommen wuppe - Es ist offensichtlich, dass in meinem Beispiel "Correctiv" durchaus auch "schwurbelt", um bei Ihrem Wortlaut zubleiben. Im Prinzip schlimmer: Es wurden nachweislich Falschbehauptungen aufstellt.
Also zieht dieses Argument nicht. 🤷‍♂️
Mir sind beide Sites im prinzip egal - ich lese beide nicht.

Dies ist nur ein Versuch auf Unausgewogenheiten hinzuweisen - was im Prinzip auch die Glaubwürdigkeit und die Wahrnehmung der Seriosität von diesem Projekt tangiert - und da gerade die "Childprotection-Liste" hier wiederholt auf den Tisch kommt, wäre es m. E. im Sinne des Erfinders, darüber noch mal zu sinieren.

> Natürlich kann es sein, dass die Listen nicht die eigene Ansicht wiederspiegeln.
> ...oder halt die _Ansichten der Listenersteller_, die Unausgewogenheit nicht erkennen kann oder will.

Wie gesagt: Ich kann die SV-PiHole sehr empfehlen - den Kinderschutz sollte man ggf. in andere Hände legen oder sich nicht drauf verlassen. That's all. Zumal Vieles von den anderen Listen abgedeckt wird. 🤷‍♂️

**Reply by Bob-Nixkoenner** [reply]
ok, da du offensichtlich die Issues gefunden hast, wieso schreibst du dann nicht einfach die anderen Seiten da rein, dass diese auch auf die Liste können? Oder bist du nur einer, der alle anderen immer kritisiert, die überhaupt etwas unternehmen? 

Um es mit deinen eigenen Worten zu sagen:

"... mit welcher Begründung" machst du das also nicht? 
Oder hast du Sorge, dass dich dann auch jemand in einem unwürdigen Ton nach einer Begründung fragt?

**Reply by RPiList** [reply]
> "Rechts oder links" - "geschwurbel oder nicht" - vollkommen wuppe - Es ist offensichtlich, dass in meinem Beispiel "Correctiv" durchaus auch "schwurbelt", um bei Ihrem Wortlaut zubleiben. Im Prinzip schlimmer: Es wurden nachweislich Falschbehauptungen aufstellt. Also zieht dieses Argument nicht. 🤷‍♂️ Mir sind beide Sites im prinzip egal - ich lese beide nicht.

Evtl. habe ich Ihr Beispiel überlesen? Wo war Ihr Beispiel mit dem Sie angeführt haben, dass Correctiv schwurbelt? 

Soweit ich das bis jetzt erkennen konnte, haben Sie überhaupt kein Beispiel gebracht. Sie hatten lediglich die Frage gestellt, warum Correctiv nicht gesperrt wurde, obwohl tischis-einblick gesperrt sei.

Also wenn Correctiv schwurbelt, dann bitte Link auf den entsprechenden Artikel liefern. Danke.

**Reply by mchawk777** [reply]
Das Niveau kann ich auch: Wo ist Ihr Beleg, dass Tichys Einblick "schwurbelt"? Schließlich ist das anscheinend Ihre Behauptung.🤷‍♂️

Anyway - als Einstieg ins Thema: https://www.youtube.com/watch?v=TXWwM9QSn2Q

**Reply by joestr** [reply]
Ich habe mir einen Artikel herausgepickt: "Warum die EU uns mit Insekten füttern will".

Der Artikel fängt mit einem Erklärungsversuch an, dass uns das Essen damit nicht mehr schmecken soll. Der Artikel geht dann über Energie, Öl, digitale Währung und endete mit einem EU-Konjunkturprogramm.

Mal abgesehen von der Artikel-Struktur, wurde in keiner einzigen Zeile darauf eingegangen, dass es nach wie vor eine **Kennzeichnungspflicht** für beigesetzte Insekten gibt.

Hier wurden wichtige Details weggelassen. Mehr als die Hälfte des Artikels widmet sich einem komplett anderen Thema.

Ein Satz sticht mir besonders ins Auge:

> Für Pollmer sind Insekten als neues Abendessen also ein weiteres Hilfsmittel für die Eurokraten, um die mitunter ja doch eigenwilligen und wenig folgsamen Bürger gefügiger zu machen und besser kontrollieren zu können.

Hier wird der "Schwurbel" frei Haus geliefert. Ich hab selten einen so dämlichen Satz gelesen.

**Reply by RPiList** [reply]
> Das Niveau kann ich auch: Wo ist Ihr Beleg, dass Tichys Einblick "schwurbelt"? Schließlich ist das anscheinend Ihre Behauptung.🤷‍♂️

Also wenn nicht bestätigt wird, dass Correctiv schwurbelt, kann es auch nicht in die Child-Protection Liste aufgenommen werden.
